src/ploigos_step_runner/step_implementers/shared/argocd_generic.py

```diff
@@ -862,3 +862,45 @@ def _argocd_get_app_manifest(
             ) from error
 
         return argocd_app_manifest_file
+
+    def _get_kube_api_uri(self):
```

Comment:
im doing this same approach for new step implemetners im creating but a cleaner way is:

```
@property
def kube_api_uri(self):
  return self.get_value('kube-api-uri')

def stuff(self):
  print(self.kube_api_uri)

```

Reply:
Didn't know this was an option! Updated my code to match.